py/desimodel/inputs/throughput.py

```diff
@@ -20,7 +20,7 @@
 
 def update(testdir=None, desi347_version=16, desi5501_version=3, desi5501_KOSI=True):
     '''
-    Update thru-\*.fits from DESI-0347 and DESI-0344
+    Update thru-[brz].fits from DESI-0347 and DESI-0344
```

Comment:
This looks fine.  I think placing the text in a rst-style code block would also work: 

```
``thru-*.fits``
```